gcd/src/main/java/org/spine3/server/storage/datastore/DsRecordStorage.java

```diff
@@ -54,10 +57,6 @@
 import static org.spine3.server.storage.datastore.DatastoreIdentifiers.keyFor;
```

Comment:
Consider having `Ds` prefix for classes like `DsIdentifiers`, `DsProperties`, etc. It's shorter and they are not supposed to be a part of public API anyway.